Tasks/ANT/task.json

```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 131,
+        "Minor": 132,
```

Comment:
Please update the "Minor" version to 133 to match the current VSTS sprint (133 goes from 3/19 to 4/6). When you build the task it will generate a task.loc.json file. Please commit that as well.

Reply:
Done. Please merge this PR if changes looks fine. If there are any other things I need to do, please tell me